README.md

```diff
@@ -13,6 +13,16 @@ files to describe single applications (eg
 [`whitehall`](https://github.com/alphagov/whitehall),
 [`frontend`](https://github.com/alphagov/frontend)) or [cross-application behaviour](features/gov_uk.feature).
 
+### Installation
+
+Smokey requires Java to be installed, because of its [use of the BrowserMob Proxy](#use-of-browsermob-proxy). [Download Java from the Oracle website](https://www.oracle.com/technetwork/java/javase/downloads/index.html) (you'll need to create an Oracle account).
```

Comment:
It's probably worth noting that these instructions are for MacOS right? And wouldn't apply if you're doing this within the Ubuntu development-vm?